metabase/metabase
# Static Row Chart formatting and legend

Epic https://github.com/metabase/metabase/issues/24759

## Changes

Adds a few improvements to the static row chart
1) Legend
2) Formatting
3) Missing features such as per series labels, a custom x-range, x-axis type

## How to verify
- create a few row charts with legend, date y-axis dimension, numeric y-axis dimension, various x-axis types, custom x-axis range, and add them to a dashboard
- send a test subscription and ensure charts look as expected

### Before
<img width="618" alt="Screen Shot 2022-10-28 at 4 47 11 AM" src="https://user-images.githubusercontent.com/14301985/198430870-8f678659-b83a-46a6-af6c-00f92871fc0d.png">
<img width="637" alt="Screen Shot 2022-10-28 at 4 46 48 AM" src="https://user-images.githubusercontent.com/14301985/198430902-9b3fc40b-d9f5-48e5-bd99-2731ce1b1b57.png">

### After
<img width="628" alt="Screen Shot 2022-10-28 at 4 46 19 AM" src="https://user-images.githubusercontent.com/14301985/198430971-8ef0808d-2c06-46e1-99ea-30ffbf4cf517.png">
<img width="622" alt="Screen Shot 2022-10-28 at 4 46 00 AM" src="https://user-images.githubusercontent.com/14301985/198431003-6ffc925a-930c-43d6-a7ee-87432cfec923.png">
